sqlmesh/core/model/definition.py

```diff
@@ -1916,3 +1917,7 @@ def _refs_to_sql(values: t.Any) -> exp.Expression:
     "hash_raw_query": exp.convert,
     "table_properties_": lambda value: value,
 }
+
+
+def is_model(node: Node) -> bool:
```

Comment:
Shouldn't this and `is_audit` be a property of `Node`?

Reply:
Is the suggestion to add to `Node` and have it return false, then override in `_Model` and `StandaloneAudit`?

Reply:
yeah, that would work too